dev-client/src/screens/HomeScreen/components/SiteMap.tsx

```diff
@@ -180,22 +185,12 @@ export const SiteMap = memo(
               paddingBottom: 0,
               cameraRef: cameraRef,
             });
-            setCalloutState({kind: 'site', siteId: feature.id as string});
+            setCalloutState(siteCallout(feature.id as string));
           }
         },
         [setCalloutState, handleClusterPress],
       );
 
-      const onTempSitePress = useCallback(
-        () =>
-          setCalloutState(
-            calloutState.kind === 'location'
-              ? {...calloutState, showCallout: true}
```

Comment:
comment: Because the `showCallout` property was unused, this callback could be safely eliminated entirely.